hera_mc/tests/test_connections.py

```diff
@@ -15,82 +12,71 @@
 
 from astropy.time import Time, TimeDelta
 
-from hera_mc import part_connect, mc, cm_handling
+from hera_mc import part_connect, mc, cm_handling, cm_utils
 from hera_mc.tests import TestHERAMC
 
 
 class TestConnections(TestHERAMC):
 
     def setUp(self):
         super(TestConnections, self).setUp()
-        # Add test parts
-        self.test_part1 = 'test_part1'
-        self.test_part2 = 'test_part2'
+        self.test_hpn = ['test_part1', 'test_part2']
         self.test_rev = 'Q'
+        self.test_mfg = 'XYZ'
+        self.test_hptype = 'vapor'
         self.test_time = Time('2017-07-01 01:00:00', scale='utc').gps
-        self.part = part_connect.Parts()
-        self.part.hpn_rev = self.test_rev
-        self.part.hptype = 'vapor'
-        self.part.manufacture_number = 'XYZ'
-        self.part.start_gpstime = self.test_time
-        self.part.hpn = self.test_part1
-        self.test_session.add(self.part)
-        self.test_session.commit()
-        self.part.hpn = self.test_part2
-        self.test_session.add(self.part)
+        self.query_time = Time('2017-08-01 01:00:00', scale='utc')
+
+        for tp in self.test_hpn:
+            self.part = part_connect.Parts()
```

Comment:
Is there any need to assign to the class attribute `self.part`, instead of just a variable `part`? I see the same pattern not just at this particular location but in most of the functions in this file.